Plugins/Detray/include/Acts/Plugins/Detray/DetrayConverter.hpp

```diff
@@ -80,7 +80,7 @@ class DetrayConverter {
     }
 
     // (2b) material grids
-    if constexpr (detray::detail::has_material_grids_v<detector_t>) {
+    if constexpr (detray::concepts::has_material_rods<detector_t>) {
```

Comment:
I think the rabbit is onto something: Are you sure this should be material rods instead of:
```suggestion
    if constexpr (detray::concepts::has_material_maps<detector_t>) {
```

Reply:
I will have a look.

Reply:
`has_material_maps` does not exist